qa/rpc-tests/wallet_parsing_amounts.py

```diff
@@ -70,19 +70,6 @@ def run_test(self):
             print(errorString)
             assert(False)
 
```

Comment:
Add a test for failure with a negative fee that is not -1.

Reply:
We have one of these? It was the one that previously took `-1` and now takes `-2`.

Reply:
Confirmed, it's in `wallet_shieldingcoinbase.py`.